### ISSUE TITLE
Customizable toolbar

### ISSUE THREAD

**Issue by rzach** [vlmantova/bookml#5]
Is it possible to customize the toolbar without hacking the XLST?

I have two specific issues:
- The `i` generates a browser popup. That may well not be accessible. It might be better to link to a separate page that authors can customize (say, using a special environment)
- It might be good to provide a way to link to an [accessibility statement](https://www.w3.org/WAI/planning/statements/) prominently and on every page so it's easy to find.

**Comment by xworld21** [comment]
> Is it possible to customize the toolbar without hacking the XLST?

No, XSLT is the only templating language supported by LaTeXML, so it has to happen there. But I can refactor the XSLT so that it's easier to customise, probably.

> * It might be good to provide a way to link to an [accessibility statement](https://www.w3.org/WAI/planning/statements/) prominently and on every page so it's easy to find.

For now, `\lxFooter{}` is the only reasonable mechanism I believe (the [Leeds guide](https://vlmantova.github.io/bookmlleeds/) has an example, for the copyright notice). I haven't implemented `\lxHeader{}` because I don't see a natural spot for it in the bookdown theme.
